body.tex

```diff
@@ -124,9 +124,6 @@ \subsection{Basic Assumptions}
 Science time is included in the subcontracts of our partners at affiliated institutions through indirect charges 
 similar to the case for NOIRLab.
 
-%When reporting actual costs (\S\ref{sec:actuals}), it may
-%be helpful to consider the following examples:
-
 In Data Production, the base assumption is that 30\% of an individual's \RO operations time (i.e. 540 hours/year for a full-time developer, 432 hours/year for a full-time scientist) are devoted to overhead for regular meetings\footnote{``Meetings'' include, for example, scheduled weekly team meetings, stand-ups, etc; major conferences or project meetings involving preparation, travel time, etc should be scheduled in advance and allocated \glspl{SP}.}, ad-hoc discussions and other interruptions.
 This work is counted as \gls{LOE}.
```

Comment:
Not obvious we should make LOE distinction - Emergent work ..

Reply:
What are you suggesting here?   To keep LOE discussion or keep "Emergent work" or something else?

Reply:
oh looks like i got distracted .. yes perhaps keep more on emergent work since LOE is a very EVMS thing.